code/modules/antagonists/changeling/powers/panacea.dm

```diff
@@ -26,8 +26,12 @@
 			C.vomit(0)
 		O.forceMove(get_turf(user))
 
+	for(var/datum/reagent/the_reagent as anything in user.reagents.reagent_list)
+		if(!the_reagent.harmful)
+			continue
+		user.reagents.remove_reagent(the_reagent.type)
+
 	user.reagents.add_reagent(/datum/reagent/medicine/mutadone, 10)
-	user.reagents.add_reagent(/datum/reagent/medicine/pen_acid, 20)
```

Comment:
Is there a particular reason that you'd kill off anatomic panacea's penacid? It's useful for healing toxins, and if anyone should have consequence free healing, it's probably changelings.

Reply:
because penacid doesnt exist